api/src/conftest.py

```diff
@@ -0,0 +1,67 @@
+"""Provides all fixtures for the testing framework
+
+TO DO:
+    * change to factory function for creation of test instance once written in app.py
+    * add database fixture
+    * add database session fixture (for scoped transactions in tests)
+    * move override settings to file
+    * set testing db to permanent file
```

Comment:
Please track your TODOs in the pull request, and not in the source code. It's easy to miss adjusting these and it's cumbersome for other readers of the code/PR to track the TODO insight a file.

Reply:
will do